src/Parser/Hunter/BeastMastery/Modules/Spells/DireBeast/DireBeastUptime.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from "common/SPELLS/index";
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import { formatPercentage } from "common/format";
+
+class DireBeastUptime extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  get percentUptime() {
+    //This calculates the uptime over the course of the encounter of Dire Beast
+    const uptime = this.combatants.selected.getBuffUptime(SPELLS.DIRE_BEAST_BUFF.id) / this.owner.fightDuration;
+    return uptime;
+  }
+  statistic() {
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.DIRE_BEAST.id} />}
+        value={`${formatPercentage(this.percentUptime)}%`}
+        label={`Dire Beast Uptime`}
+        tooltip={`If this tooltip shows over 100% uptime, it's because you on average had more than 1 Dire Beast buff active at all times. In actuality you had an average of ${(this.percentUptime).toFixed(2)} up throughout the fight.`}
```

Comment:
Why does the tooltip say `In actuality you had an average of ${(this.percentUptime).toFixed(2)}` when this.percentUptime is also the stat shown in the value? I don't understand what the difference is here.

Reply:
One shows an actual percentage number (say 170%), and one shows a regular number which in this case would be 1.7. 

It should probably say ``you had an average of X Dire Beasts up throughout the fight.``